### ISSUE TITLE
The Smooth l1 Loss of Key-point Locations

### ISSUE THREAD

**Issue by mariolew** [tfwu/FaceDetection-ConvNet-3D#3]
Hi, thanks for the work.

I've just found that the code is not doing the same way as the paper said. 
For instance, The **Smooth l1 Loss of Key-point Locations** is not the same. In the paper, only the m predicted labels contribute to the loss. In the code, since the **_proj_label_** are the set to be all zeros besides 9 \* m locations around m key points, every location will contribute to the loss.

Can anyone explain this for me?

**Comment by tensorboy** [comment]
Did you successfully train the model VGG16_813001.params ?
I stuck on here as I'm not familiar with mxnet.

**Comment by mariolew** [comment]
@tensorboy No, I still don't quite understand the paper. I mean I don't quite understand how the labels are formed, how the losses are computed and how the proposals are generated. Maybe you've understood the paper?
